### PR TITLE
Add WAKE-REM-NREM classifiers

### PR DESCRIPTION
**Pull request by hofaflo**

### CODE REVIEW

**Comment by hofaflo** [comment]
[PhysioNet](https://physionet.org/) appears to be down right now which causes the tests to fail. Apart from that, this is ready for review @cbrnr :)

File: doc/source/classification.md [hunk 1]
```diff
@@ -8,7 +8,19 @@ Note that the performance metrics are averages, so a classifier might perform mu
...
 |-|-|-|-|-|-|-|-|
-|`ws-gru-mesa`|WAKE-SLEEP|[GRU](https://en.wikipedia.org/wiki/Gated_recurrent_unit)|MESA (1971)|SHHS (1000)|0.83|0.60|[Link](https://github.com/cbrnr/sleepecg/blob/main/examples/classifiers/ws_gru_mesa.py)|
+|`ws-gru-mesa`|WAKE-SLEEP|[GRU](https://en.wikipedia.org/wiki/Gated_recurrent_unit)|MESA (1970)|SHHS (1000)|0.83|0.60|[Link](https://github.com/cbrnr/sleepecg/blob/main/examples/classifiers/ws_gru_mesa.py)|
+|`wrn-gru-mesa`|WAKE-REM-NREM|[GRU](https://en.wikipedia.org/wiki/Gated_recurrent_unit)|MESA (1970)|SHHS (1000)|0.75|0.54|[Link](https://github.com/cbrnr/sleepecg/blob/main/examples/classifiers/wrn_gru_mesa.py)|
+|`wrn-gru-mesa-weighted`|WAKE-REM-NREM|[GRU](https://en.wikipedia.org/wiki/Gated_recurrent_unit)|MESA (1970)|SHHS (1000)|0.70|0.53|[Link](https://github.com/cbrnr/sleepecg/blob/main/examples/classifiers/wrn_gru_mesa_weighted.py)|
+
+
+### Notes
+As REM is much less common than NREM and WAKE, the current models show limited performance in WAKE-REM-NREM classification.
+Using no sample weights, the `wrn-gru-mesa` model identifies 64% of actual REM stages as NREM, as shown in the confusion matrix below (left).
```
Comment (cbrnr) [inline doc/source/classification.md:18]:
```suggestion
Using no sample weights, the `wrn-gru-mesa` model identifies 64% of actual REM stages as NREM as shown in the confusion matrix below (left).
```

File: doc/source/classification.md [hunk 2]
```diff
@@ -8,7 +8,19 @@ Note that the performance metrics are averages, so a classifier might perform mu
...
+With sample weights inversely proportional to class frequency in `wrn-gru-mesa-weighted`, the confusion matrix (right) is more symmetrical.
+While the values normalized over "true" classes (given in parentheses) look promising, note that the number of actual NREM stages identified as REM is larger than the number of correctly identified REM stages (i.e. the model has low precision for REM).
```
Comment (cbrnr) [inline doc/source/classification.md:20]:
```suggestion
While the values normalized over true classes (given in parentheses) look promising, the number of actual NREM stages identified as REM is larger than the number of correctly identified REM stages (i.e. the model has low precision for REM).
```

**Review by cbrnr — APPROVED** [review]
Besides my minor nitpicks LGTM!

**Pull request merged**